walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -82,6 +89,44 @@
       );
     }
 
-    return possibleTimes;
+    // If there were no mandatory attendees or there are no optional attendees, return now.
```

Comment:
This `query` function is quite long. Could you refactor into smaller components for better readability.

Reply:
I attempted to refactor!